assets/css/dup/dup_screen_page.scss

```diff
@@ -3,9 +3,32 @@
   grid-template-columns: 1fr 1fr 1fr;
 }
 
-.simulation-page {
+.simulation-screen-centering-container {
+  display: flex;
+  justify-content: center;
   background: #3d5366;
+}
+
+.simulation-screen-scrolling-container {
+  &::-webkit-scrollbar {
```

Comment:
I think we need to be careful styling browser specific CSS because it won't necessarily work in all browsers. For example, `webkit` is only guaranteed to work for Chrome and Safari. I use Firefox as my browser and none of these styles are showing up.

Reply:
I will say that the scrollbar looks great when using Chrome! And `::webkit-scrollbar` is supported by all browsers except Firefox. After some quick googling, there doesn't seem to be a way to support this on FF, so we may just leave it alone for that one browser. Will leave that up to you and design.

Reply:
Turns out that Paul H interpreted from the mocks that "there should be a scrollbar" and not "there should be a custom scrollbar". I didn't notice that on the Asana ticket until after I already customized the scrollbar 😅  So I checked with him in Slack, and he says this is fine